json_serializable/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.6
+
+* Added support for `JsonSerializable.disallowUnrecognizedKeys`.
```

Comment:
Also note that `fromJson` functions now use block syntax

Reply:
done